apps/meteor/client/views/room/MessageList/lib/isMessageSequential.ts

```diff
@@ -28,5 +28,9 @@ export const isMessageSequential = (current: IMessage, previous: IMessage | unde
 		return false;
 	}
 
+	if (isThreadMessage(previous) && previous.tshow) {
+		return false;
+	}
```

Comment:
please break this test into another function, in that way its not possible to use this function inside threads for example